debezium-server/debezium-server-core/src/main/java/io/debezium/server/DebeziumServer.java

```diff
@@ -122,6 +123,10 @@ else if (beans.size() > 1) {
             props.setProperty("transforms", transforms.get());
             configToProperties(config, props, PROP_TRANSFORMS_PREFIX, "transforms.");
         }
+        if ("pubsub".equals(name)) {
```

Comment:
How about having a method like `supportsTombstoneEvents()` on the `ChangeConsumer` contract instead of hard-wiring this?